ruuvi_examples/ruuvi_firmware/bluetooth_application_config.h

```diff
@@ -8,4 +8,9 @@
 #define APPLICATION_ADV_INTERVAL        1000                            /**< ms **/
 #define APP_TX_POWER                    4                               /**< dBm **/
 
+// milliseconds until main loop timer function is called. Other timers can bring
+// application out of sleep at higher (or lower) interval.
+#define MAIN_LOOP_INTERVAL_URL  500u 
```

Comment:
URL interval is used with main loop timer to decide when sensors are read, decreasing this to 500u would cause extra power consumption. 

You could add ADVERTISING_INTERVAL_URL 500u and use it instead